### PR TITLE
eloquence syntax

### PR DESCRIPTION
**Pull request by macMikey**
syntax for both eloquence program files and eloquence dialog files
https://eloquence.marxmeier.com/

### CODE REVIEW

**Comment by braver** [comment]
Please use the checklist in the PR template to tell us about your package.

**Comment by macMikey** [comment]
is this what you mean? i did not see the template, so i copied the template from another PR

* [x]  I'm the package's author and/or maintainer.
* [x]  I have have read [the docs](https://packagecontrol.io/docs/submitting_a_package).
* [x]  I have tagged a release with a [semver](https://semver.org) version number.
* [x]  My package repo has a description and a README describing what it's for and how to use it.
* [x]  My package doesn't add context menu entries. *
* [x]  My package doesn't add key bindings. **
* [x]  Any commands are available via the command palette.
* [x]  Preferences and keybindings (if any) are listed in the menu and the command palette, and open in split view.
* [x]  If my package is a syntax it doesn't also add a color scheme. ***
* [x]  If my package is a syntax it is named after the language it supports (without suffixes like "syntax" or "highlighting").
* [ ]  I use [.gitattributes](https://www.git-scm.com/docs/gitattributes#_export_ignore) to exclude files from the package: images, test files, sublime-project/workspace.


there are no extraneous files or directories in the repo, so, there is no need for .gitattributes

**Comment by braver** [comment]
Was this intended to be a syntax file too? https://github.com/macMikey/eloquence-sublime-text-syntax/blob/main/dlg-sublime-syntax.yaml

**Comment by braver** [comment]
In captures, you don't need to put quotes around the indexes. Also please ensure all scopes end with `.eq` (matching "source`.eq`"). Also please double check [the guidelines](https://www.sublimetext.com/docs/scope_naming.html), you might be able to specify some of those keywords. E.g.:

```yml
captures:
        1: keyword.control.eq
        2: entity.name.function.eq
```

File: repository/e.json [hunk 1]
```diff
@@ -735,6 +735,17 @@
 				}
 			]
 		},
+		{
+			"name": "Eloquence Syntax",
```
Comment (braver) [inline repository/e.json:739]: > If my package is a syntax it is named after the language it supports (without suffixes like "syntax" or "highlighting").

.... 🧐 


```suggestion
			"name": "Eloquence",
```

**Review by braver — CHANGES_REQUESTED** [review]

**Comment by macMikey** [comment]
> Was this intended to be a syntax file too? https://github.com/macMikey/eloquence-sublime-text-syntax/blob/main/dlg-sublime-syntax.yaml

there are two types of eloquence files: eloquence program files (.eq), and eloquence dialog files (.dlg). they are supposed to be included.

**Comment by macMikey** [comment]
thanks for your patience, i'm a package n00b

**Comment by braver** [comment]
You'll probably want to rename [dlg-sublime-syntax.yaml](https://github.com/macMikey/eloquence-sublime-text-syntax/blob/main/dlg-sublime-syntax.yaml) to `dlg.sublime-syntax` then. 

> thanks for your patience, i'm a package n00b

No worries, that's what we're here for 😉

**Comment by macMikey** [comment]
> You'll probably want to rename [dlg-sublime-syntax.yaml](https://github.com/macMikey/eloquence-sublime-text-syntax/blob/main/dlg-sublime-syntax.yaml) to `dlg.sublime-syntax` then.
> 
> > thanks for your patience, i'm a package n00b
> 
> No worries, that's what we're here for 😉

done

**Comment by braver** [comment]
Alright, main thing then remains that all scopes need to end with the identifier of the syntax, ie. `keyword.dlg` and `keyword.eq`, etc. This makes it possible for package developers to target your specific syntax.

**Comment by macMikey** [comment]
so: in the .dlg syntax file, all of the scopes end in .dlg and in the .eq file, all the scopes end in .eq?

**Comment by braver** [comment]
Yes. Basically your base scope for a syntax is source.XX and then you make sure all the scopes in that syntax also end with .XX

**Comment by braver** [comment]
Almost!

source.dlg.eq should still be source.dlg

Also remember that your captures also assign scopes, so you need to cover those as well.

Have a look at some example syntaxes like https://github.com/sublimehq/Packages/blob/master/CSS/CSS.sublime-syntax#L3216C1-L3224C43

**Comment by macMikey** [comment]
* source.dlg updated
* captures updated

thanks for all the help.

**Comment by braver** [comment]
Looks good!

**Comment by macMikey** [comment]
thanks for all the help.
package-control n00b...out

**Comment by braver** [comment]
🫡

**Pull request merged**